website/blog/2021-07-28-release-apache-apisix-2.8.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Apache APISIX 2.8 正式发布,带来更多新功能！"
+author: spacewander
+authorURL: "https://github.com/spacewander"
+authorImageURL: "https://avatars.githubusercontent.com/u/4161644?v=4"
+---
+> [@spacewander](https://github.com/spacewander), Core developer of Apache APISIX from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+<!--truncate-->
+
+Apache APISIX 2.8 版本正式发布！🎉 这个版本有 30+ 开发者参与，共提交了 100+ PR，支持了**1 个新功能，1 个新体验，2 个新插件，2 个新玩法**，快速阅读了解 2.8 版本的新特性吧！
+
+👇👇👇
+
+## Release Notes
+
+### 新功能：独立的 Keepalive 连接池
+
+从[2.7 版本](https://apisix.apache.org/blog/2021/06/29/release-apache-apisix-2.7)开始添加 Apache APISIX 自己的补丁和 Nginx C 模块，增强原生 Nginx 的功能，希望能够动态设置越来越多的 Nginx 配置。在发布的最新版本中，Apache APISIX 已经支持在 Upstream 级别上配置独立的 Keepalive 连接池。
+
+目前包含了以下功能：
+
+- 动态设置 mTLS
+- 动态设置 client_max_body_size
+- Upstream 的 keepalive（2.8 新功能）
+- gzip （2.8 新插件）
+
+在 Apache APISIX 后续版本中，我们也会陆续允许下面的 Nginx 配置能够被动态设置：
+
+- real_ip
+- proxy_max_temp_file_size
+- ……
+
+Upstream 的配置举例：
+
+```JSON
+{
+    "id": "backend",
+    "nodes": {"host:80": 100},
+    "type":"roundrobin",
+    "keepalive_pool": {
+        "size": 4,
+        "idle_timeout": 8,
+        "requests": 16
+    }
+}
+```
+
+### 新体验：stream 代理功能增强
+
+在 2.8 版本中，把 ip-restriction 和 limit-conn 两个插件从 HTTP 部分的移植到了 stream 部分，这么做的好处是丰富网关在 stream 代理的功能，增加对上游服务的安全性保障。
```

Comment:
Add hyperlinks for these two plugins?

Reply:
good idea！Can you tell me where are the hyperlinks？I'm not familiar with this part.

Reply:
* ip-restriction: http://apisix.apache.org/docs/apisix/plugins/ip-restriction/
* limit-conn: http://apisix.apache.org/docs/apisix/plugins/limit-conn/